### PR TITLE
TMEDIA-378 - Top table list stories

### PR DESCRIPTION
**Pull request by matthewroach**
## Description
Add story book stories for Top Table List Block

## Jira Ticket
- [TMEDIA-378](https://arcpublishing.atlassian.net/browse/TMEDIA-378)

View in chromatic - https://www.chromatic.com/build?appId=5f91e4721ccaba0022a10782&number=764

### CODE REVIEW

**Comment by JackHowa** [comment]
was this a side effect of some shared styles? @matthewroach 
Screen Shot 2021-08-17 at 13 57 24

https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=611bbab3266fc6003c37dcec
Screen Shot 2021-08-17 at 13 58 01

**Comment by JackHowa** [comment]
for top table list components, it looks great. nice work especially with different options

**Comment by matthewroach** [comment]
I am not 100% sure on how other changes were highlighted, as the only change was me adding a story file. The only thing I can think of is its a slight issue with rendering on chromatic's side

**Comment by matthewroach** [comment]
The only other explanation is the baseline has not been correctly updated in the past - @JackHowa

File: blocks/top-table-list-block/features/top-table-list/default.scss [hunk 1]
```diff
@@ -1,9 +1,11 @@
-.xl-large-promo {
-  .xl-promo-headline {
-    @media screen and (min-width: map-get($grid-breakpoints, 'md')) {
-      margin-bottom: map-get($spacers, 'xs');
-    }
-  }
+.top-table-list-section-extra_large {
```
[the remark this answers is not in the document]
Comment (matthewroach) [reply blocks/top-table-list-block/features/top-table-list/default.scss:1]: Not a huge issue - we can leave as is for now

**Pull request merged**